examples/all-clusters-app/esp32/README.md

```diff
@@ -160,73 +174,60 @@ your network configuration. To erase it, simply run.
-    application and is configurable through menuconfig.
+    $ chip-tool pairing ble-wifi ${NODE_ID_TO_ASSIGN} ${SSID} ${PASSWORD} 0 20202021 3840
 
-         - chip-device-ctrl > ble-scan
-         - chip-device-ctrl > connect -ble 3840 20202021 135246
+or equivalently:
 
-         Parameters:
-         1. Discriminator: 3840 (configurable through menuconfig)
-         2. Setup-pin-code: 20202021 (configurable through menuconfig)
-         3. Node ID: Optional.
-            If not passed in this command, then it is auto-generated by the controller and displayed in the output of connect.
-            The same value should be used in the next commands.
-            We have chosen a random node ID which is 135246.
+    $ chip-tool pairing ble-wifi 17 hex:787980 hex:736563726574 0 20202021 3840
 
--   Add credentials of the Wi-Fi network you want the ESP32 to connect to, using
-    the `AddOrUpdateWiFiNetwork` command and then enable the ESP32 to connect to
-    it using `EnableWiFiNetwork` command. In this example, we have used
-    `TESTSSID` and `TESTPASSWD` as the SSID and passphrase respectively.
+#### Pair a device over IP
 
-         - chip-device-ctrl > zcl NetworkCommissioning AddOrUpdateWiFiNetwork 135246 0 0 ssid=str:TESTSSID credentials=str:TESTPASSWD breadcrumb=0
+The command below will discover devices with long discriminator 3840 and try to
+pair with the first one it discovers using the provided setup code.
 
-         - chip-device-ctrl > zcl NetworkCommissioning ConnectNetwork 135246 0 0 networkID=str:TESTSSID breadcrumb=0
+    $ chip-tool pairing onnetwork-long ${NODE_ID_TO_ASSIGN} 20202021 3840
 
--   Close the BLE connection to ESP32, as it is not required hereafter.
+The command below will discover devices based on the given QR code (which
+devices log when they start up) and try to pair with the first one it discovers.
 
-         - chip-device-ctrl > close-ble
+    $ chip-tool pairing qrcode ${NODE_ID_TO_ASSIGN} MT:#######
 
--   Resolve DNS-SD name and update address of the node in the device controller.
+### Forget the currently-commissioned device
 
-         - chip-device-ctrl > resolve 135246
+    $ chip-tool pairing unpair
 
 ### Cluster control
 
--   After successful commissioning, use the OnOff cluster commands to control
-    the OnOff attribute. This allows you to toggle a parameter implemented by
-    the device to be On or Off.
+### onoff
 
-    `chip-device-ctrl > zcl OnOff Off 135246 1 1`
+To use the Client to send Matter commands, run the built executable and pass it
+the target cluster name, the target command name as well as an endpoint id.
 
--   Use the LevelControl cluster commands to control the CurrentLevel attribute.
-    This allows you to control the brightness of the led.
+The endpoint id must be between 1 and 240.
 
-    `chip-device-ctrl > zcl LevelControl MoveToLevel 135246 1 1 level=10 transitionTime=0 optionMask=0 optionOverride=0`
+    $ chip-tool onoff on 12344321 1
```

Comment:
You have used `${NODE_ID_TO_ASSIGN}` above and directly use 12344321 here

Reply:
done